README.md

```diff
@@ -25,7 +25,7 @@
 [dependabot]: https://api.dependabot.com/badges/status?host=github&repo=mohsen1/service-worker-loader
 [dependabot-url]: https://dependabot.com/
 
-Modern ServiceWorker loader for [Webpack](https://webpack.js.org).
+Modern ServiceWorker loader for [Webpack](https://webpack.js.org). Takes a JS file, emits it separately from the bundle, and returns a function to register the file as a service worker.
```

Comment:
> Takes a ~~JS~~ file

It can take not only JS file.

I think better variant is:

> Modern service worker loader for [Webpack](https://webpack.js.org): takes a file, emits it separately from the bundle, and exports a function to register the file as a service worker.